src/OptionSettlementEngine.sol

```diff
@@ -172,12 +169,16 @@ contract OptionSettlementEngine is ERC1155, IOptionSettlementEngine {
     //  Public/External Views
     //////////////////////////////////////////////////////////////*/
 
+    //
+    // Option Information
+    //
+
     /// @inheritdoc IOptionSettlementEngine
-    function option(uint256 tokenId) external view returns (Option memory optionInfo) {
-        (uint160 optionKey,) = _decodeTokenId(tokenId);
+    function option(uint256 optionId) external view returns (Option memory optionInfo) {
```

Comment:
This is incorrect, it actually takes a tokenId, not an optionId, and returns information about the option type of that token, even if it's a claim. The naming was purposeful.

Reply:
Okay, good point. Clarifying this in NatSpec